### ISSUE TITLE
Exception while trying to run SRL 

### ISSUE THREAD

**Issue by eme-ele** [clir/clearnlp-guidelines#9]
I am having the following trace when attempting to use decoding for SRL on a given .txt file
Installation was successful (I have everything on my classpath) and the log4j.properties has been added to the classpath. Any information on how to deal with this?
```
java -Xmx5g -XX:+UseConcMarkSweepGC edu.emory.clir.clearnlp.bin.NLPDecode -mode srl -c config_decode_srl.xml -i data/myfile.txt
```

```java
java.io.IOException: Stream closed
	at java.io.BufferedInputStream.getInIfOpen(BufferedInputStream.java:159)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:286)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readFully(DataInputStream.java:169)
	at org.tukaani.xz.SingleXZInputStream.initialize(Unknown Source)
	at org.tukaani.xz.SingleXZInputStream.<init>(Unknown Source)
	at org.tukaani.xz.XZInputStream.<init>(Unknown Source)
	at org.tukaani.xz.XZInputStream.<init>(Unknown Source)
	at edu.emory.clir.clearnlp.component.utils.NLPUtils.getObjectInputStream(NLPUtils.java:173)
	at edu.emory.clir.clearnlp.component.utils.NLPUtils.getNERecognizer(NLPUtils.java:126)
	at edu.emory.clir.clearnlp.bin.NLPDecode.getComponents(NLPDecode.java:180)
	at edu.emory.clir.clearnlp.bin.NLPDecode.decode(NLPDecode.java:94)
	at edu.emory.clir.clearnlp.bin.NLPDecode.<init>(NLPDecode.java:74)
	at edu.emory.clir.clearnlp.bin.NLPDecode.main(NLPDecode.java:236)
java.lang.NullPointerException
	at edu.emory.clir.clearnlp.component.AbstractStatisticalComponent.load(AbstractStatisticalComponent.java:157)
	at edu.emory.clir.clearnlp.component.AbstractStatisticalComponent.initDecode(AbstractStatisticalComponent.java:126)
	at edu.emory.clir.clearnlp.component.AbstractStatisticalComponent.<init>(AbstractStatisticalComponent.java:99)
	at edu.emory.clir.clearnlp.component.mode.ner.AbstractNERecognizer.<init>(AbstractNERecognizer.java:61)
	at edu.emory.clir.clearnlp.component.mode.ner.EnglishNERecognizer.<init>(EnglishNERecognizer.java:52)
	at edu.emory.clir.clearnlp.component.utils.NLPUtils.getNERecognizer(NLPUtils.java:119)
	at edu.emory.clir.clearnlp.component.utils.NLPUtils.getNERecognizer(NLPUtils.java:126)
	at edu.emory.clir.clearnlp.bin.NLPDecode.getComponents(NLPDecode.java:180)
	at edu.emory.clir.clearnlp.bin.NLPDecode.decode(NLPDecode.java:94)
	at edu.emory.clir.clearnlp.bin.NLPDecode.<init>(NLPDecode.java:74)
	at edu.emory.clir.clearnlp.bin.NLPDecode.main(NLPDecode.java:236)
	Exception in thread "main" java.lang.NullPointerException
	at edu.emory.clir.clearnlp.component.mode.ner.NERState.init(NERState.java:59)
	at edu.emory.clir.clearnlp.component.mode.ner.NERState.<init>(NERState.java:52)
	at edu.emory.clir.clearnlp.component.mode.ner.AbstractNERecognizer.process(AbstractNERecognizer.java:98)
	at edu.emory.clir.clearnlp.bin.NLPDecode.process(NLPDecode.java:163)
	at edu.emory.clir.clearnlp.bin.NLPDecode.process(NLPDecode.java:132)
	at edu.emory.clir.clearnlp.bin.NLPDecode.decode(NLPDecode.java:108)
	at edu.emory.clir.clearnlp.bin.NLPDecode.<init>(NLPDecode.java:74)
	at edu.emory.clir.clearnlp.bin.NLPDecode.main(NLPDecode.java:236)
```

**Comment by eme-ele** [comment]
By looking at the traceback I see that the ner module was being accessed and added `<ner>general-en-ner.xz</ner>` to the models in the configuration file and I managed to reduce the error to:
```java
Exception in thread "main" java.lang.NullPointerException
	at edu.emory.clir.clearnlp.component.mode.ner.NERState.init(NERState.java:59)
	at edu.emory.clir.clearnlp.component.mode.ner.NERState.<init>(NERState.java:52)
	at edu.emory.clir.clearnlp.component.mode.ner.AbstractNERecognizer.process(AbstractNERecognizer.java:98)
	at edu.emory.clir.clearnlp.bin.NLPDecode.process(NLPDecode.java:163)
	at edu.emory.clir.clearnlp.bin.NLPDecode.process(NLPDecode.java:132)
	at edu.emory.clir.clearnlp.bin.NLPDecode.decode(NLPDecode.java:108)
	at edu.emory.clir.clearnlp.bin.NLPDecode.<init>(NLPDecode.java:74)
	at edu.emory.clir.clearnlp.bin.NLPDecode.main(NLPDecode.java:236)
```
I suggest reviewing the configuration files. However, the ner module keeps giving me errors when using the srl mode. Any suggestions?

**Comment by eme-ele** [comment]
I fixed it by using clearnlp-3.2.0.jar, installation guidelines and the link to the .jar should be updated with this!

**Comment by jdchoi77** [comment]
Thank you and great that it got fixed :)

Jinho
